README.md

```diff
@@ -26,7 +29,7 @@ You can learn more about the config options [here](https://github.com/IdentityMo
 
 1. Make sure Node.js is installed, installation instructions are available at [node.js](https://nodejs.org/en/)
 2. Clone the git repository
-3. Open a command prompt and navigate to this folder (basic_samples/Authentication/AuthorizationCodeFlow/JavaScript)
+3. Open a command prompt and navigate to this folder (sample-adh-authentication_authorization-nodejs)
```

Comment:
I'm not sure we need this line. it was necessary when cloning the repo gave you dozens of samples, but since it's one sample per repo now, this line really feels implied and could be deleted. thoughts?

Reply:
Yes that is a good point, I'll remove it